frontend/src/components/ChapterMap.tsx

```diff
@@ -1,99 +1,224 @@
-import L from 'leaflet'
-import React, { useEffect, useMemo, useRef } from 'react'
-import 'leaflet/dist/leaflet.css'
-import 'leaflet.markercluster/dist/MarkerCluster.css'
-import 'leaflet.markercluster/dist/MarkerCluster.Default.css'
-import 'leaflet.markercluster'
-import { GeoLocDataAlgolia, GeoLocDataGraphQL } from 'types/chapter'
-
-const ChapterMap = ({
+import L from 'leaflet';
+import React, { useEffect, useMemo, useRef, useCallback } from 'react';
+import 'leaflet/dist/leaflet.css';
+import 'leaflet.markercluster/dist/MarkerCluster.css';
+import 'leaflet.markercluster/dist/MarkerCluster.Default.css';
```

Comment:
Also, please run checks before submitting code with `make check-test-all`. There are many unnecessary changes, like added `;` for example. This makes it hard to review, since we have to dig through irrelevant code. 

We do not use `;` at the end of the lines. Running `make check-test-all` will clean this up for you.